### ISSUE TITLE
Object of type datetime is not JSON serializable

### ISSUE THREAD

**Issue by yubarajshrestha** [MasoniteFramework/orm#475]
There's an issue with dates if you use QueryBuilder.

**To Reproduce**
Add few records into any model. For now let's, use "User" model. There are two ways to get the users list from database, 1st one is using `User` model 2nd one is using `QueryBuilder`. Querying using Model works perfectly fine but with QueryBuilder it gives error.

<img width="1680" alt="Screen Shot 2021-07-07 at 6 40 34 PM" src="https://user-images.githubusercontent.com/7955362/124762665-e0fe3300-df52-11eb-987c-526b3adf61c0.png">

**Working Code**
```python
users = User.get()
return users
```

**Not Working**
```python
users = QueryBuilder().table("users").get()
return users
```

There are times when we have to use QueryBuilder, for which date parsing in json is giving issue. `__dates__` in model is also not helping.

**Comment by josephmancuso** [comment]
Fixed in version 1.0.53

**Comment by yubarajshrestha** [comment]
> Fixed in version 1.0.53

It's not working... Still getting same issue.
